src/evaluator.rs

```diff
@@ -83,50 +83,63 @@ impl<'src, 'run> Evaluator<'src, 'run> {
     Ok(self.scope.value(name).unwrap())
   }
 
+  /// A place for adding list operators in the future.
```

Comment:
This is a little confusing to me, since it adds some indirection to follow to figure out what's going on. I think we should probably remove it, and just use `evaluate_expression`, and we can add it back later when we need it.